pkg/dbmate/db.go

```diff
@@ -72,7 +72,7 @@ func New(databaseURL *url.URL) *DB {
 // GetDriver initializes the appropriate database driver
 func (db *DB) GetDriver() (Driver, error) {
 	if db.DatabaseURL == nil || db.DatabaseURL.Scheme == "" {
-		return nil, errors.New("invalid url")
+		return nil, errors.New("invalid url, have you set your --url flag or DATABASE_HOST environment variable?")
```

Comment:
Thanks! I think you mean DATABASE_URL?

Reply:
I do! Apologies. This is what I get for trying to do a "quick" quality of life fix while talking to somebody!
Pushed up a fix.